clang missing limits

while building the clang package get undefined reference to std limit functions.
patch attached.
[0013-missing-limits.zip](https://github.com/msys2/MSYS2-packages/files/9147259/0013-missing-limits.zip)

--- a/clang/0013-missing-limits.patch
+++ b/clang/0013-missing-limits.patch
@@ -1,0 +1,11 @@
+diff -Naur a/utils/benchmark/src/benchmark_register.h b/utils/benchmark/src/benchmark_register.h
+--- a/utils/benchmark/src/benchmark_register.h	2020-10-07 12:10:48.000000000 +0200
++++ b/utils/benchmark/src/benchmark_register.h	2022-07-20 08:33:14.303536000 +0200
+@@ -2,6 +2,7 @@
+ #define BENCHMARK_REGISTER_H
+ 
+ #include <vector>
++#include <limits>
+ 
+ #include "check.h"
+ 

--- a/clang/PKGBUILD
+++ b/clang/PKGBUILD
@@ -15,7 +15,7 @@ _realname=clang
 pkgbase=${_realname}
 pkgname=("${_realname}" "lld" "llvm")
 pkgver=11.0.0
-pkgrel=4
+pkgrel=5
 pkgdesc="C language family frontend for LLVM"
 arch=('i686' 'x86_64')
 url="https://llvm.org/"
@@ -39,6 +39,7 @@ source=(${_url}/clang-${pkgver}.src.tar.xz{,.sig}
         "0010-mbig-obj-for-all.patch"
         "0011-Msysize.patch"
         "0012-Add-minimal-msys-target.patch"
+        "0013-missing-limits.patch"
         "0101-Allow-build-static-clang-library-for-mingw.patch"
         "0102-fix-libclang-name-for-mingw.patch"
         "0105-build-libclang-cpp-fix.patch"
@@ -65,6 +66,7 @@ sha256sums=('0f96acace1e8326b39f220ba19e055ba99b0ab21c2475042dbc6a482649c5209'
             '1c9efccd40a0e7834c3aa9d819aa25cfdd2cec389d1bd3e8a89bc9ff670a0129'
             '3042a28691050688b0d9c9aebe803d2412c0effaf5c57b5a4b18144df91f224e'
             'fb37e32d843c817aaa57a15f2f65014c38eb80bfe2af7b84fe5c6f356eea836a'
+            '1488c3032a23e7fc79e983ae286326bdfa3590cdbd25ea74c8dda6ad59a83c52'
             'ddc43bc16f5edaf5290adabd71ce6a8a593fc3f824d7d99acd247bcce88aaf03'
             '01b029f2a21bd998ce374a90d41d214c891dfbb611dfbd9ca147517cd2c228ea'
             'a60f7328d84628a56a9f626e4dc26ffd0c35292c79eeba62ac3d4f25aef2fe5c'
@@ -99,7 +101,8 @@ prepare() {
       "0009-empty-target-prefix-only-msvc.patch" \
       "0010-mbig-obj-for-all.patch" \
       "0011-Msysize.patch" \
-      "0012-Add-minimal-msys-target.patch"
+      "0012-Add-minimal-msys-target.patch" \
+      "0013-missing-limits.patch"
 
   cd "${srcdir}/clang-${pkgver}.src"
   apply_patch_with_msg \

